### DISCUSSION TITLE
Add active tab border

### DISCUSSION THREAD

**Discussion by lampewebdev** [ykrx/Lightbulb#3] [Ideas]
I have added the following setting:

```json
  "workbench.colorCustomizations": {
    "tab.activeBorder": "#ffcf6f"
  },
```
How it looks:
<img width="450" alt="image" src="https://user-images.githubusercontent.com/44268468/200931433-5b7c5bfb-d94f-4533-9494-b92e46e93f72.png">

It adds the bottom line.  
For me this makes it easier to understand which tab is active right now.

**Comment by ykrx** [comment]
<img width="752" alt="Screenshot 2022-11-09 at 3 37 07 PM" src="https://user-images.githubusercontent.com/43590347/200938215-aa08495b-7086-4b3d-9f34-313c746effd3.png">

Yeah I considered using the bottom border but then it breaks the illusion of the editor blending into the active tab's backgroundColor. My idea was to match the consistency with activityBar's left edge border, but it only looks good when the title bar is displayed inline (via CustomizeUI; there might be ways to thicken the top border, but haven't looked into it yet). I wish VSCode allowed tab title styling overrides for active tabs, I think that the currently active tab title should be the primary yellow color (which it is, but only for files not under source control — otherwise VSCode sets the tab title color based on the file's git status). I tried setting modified git decorations to the primary yellow color but then it gets really chaotic in busy workspaces, so still playing around with ideas.

Also wish we could override that annoying, gray window border; would add more visual contrast between the active edge borders. If anyone knows of ways to mess with it feel free to let me know!

---

### Aside:
Would also be cool if Lightbulb applied all of the recommended setup settings on its own, but I'm not sure if color themes are allowed that much scope.
